dataclients/kubernetes/path_test.go

```diff
@@ -79,10 +79,7 @@ func TestPathMatchingModes(t *testing.T) {
 	defer api.Close()
 
 	setIngressWithPath := func(p string, annotations ...string) {
-		i := testIngress(
-			"namespace1", "ingress1", "service1", "", "", "", "", "", "", definitions.BackendPort{Value: 8080}, 1.0,
-			testRule("www.example.org", testPathRule(p, "service1", definitions.BackendPort{Value: 8080})),
-		)
+		i := testIngress("namespace1", "ingress1", "service1", "", "", "", "", "", "", map[string]string{}, definitions.BackendPort{Value: 8080}, 1.0, testRule("www.example.org", testPathRule(p, "service1", definitions.BackendPort{Value: 8080})))
```

Comment:
If it's a noop then please don't change the code

Reply:
oops, I guess I accidentally formatted it in IDE 🥺